tests/data/test_rosbag.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+from pathlib import Path
+
+from flowcean.environments.rosbag import RosbagEnvironment
+
+
+class TestRosbagEnvironment(unittest.TestCase):
+    def test_rosbag_loader_multiple_topics(self) -> None:
+        path = Path("examples/ros_offline/test_rosbag")
+        environment = RosbagEnvironment(
+            path=path,
+            topics={
+                "/j100_0000/amcl_pose": [
+                    "pose.pose.position.x",
+                    "pose.pose.position.y",
+                ],
+                "/j100_0000/platform/odom": [
+                    "pose.pose.position.x",
+                    "pose.pose.position.y",
+                ],
+            },
+        )
+        environment.load()
+        data = environment.get_data()
+
+        assert data.columns == [
+            "j100_0000/amcl_pose.time",
+            "j100_0000/amcl_pose.pose.pose.position.x",
+            "j100_0000/amcl_pose.pose.pose.position.y",
+            "j100_0000/platform/odom.time",
+            "j100_0000/platform/odom.pose.pose.position.x",
+            "j100_0000/platform/odom.pose.pose.position.y",
+        ]
```

Comment:
This is a pretty weak test, in my opinion - it only really tests whether the code runs, without making strong assertions about any results. We can keep it for now, but should extend it by something stronger soon. Maybe recording an artifical ros bag would be an idea which just contains a sine wave or something similar ?